test/provider_devdax_memory.cpp

```diff
@@ -304,6 +304,34 @@ TEST_P(umfProviderTest, get_name) {
     ASSERT_STREQ(name, "DEVDAX");
 }
 
+TEST(DevDaxProviderName, custom_name) {
+    auto params_handle = create_devdax_params();
+    if (!params_handle.get()) {
+        GTEST_SKIP() << "devdax params unavailable";
+    }
+
+    const char *custom = "my_devdax";
+    ASSERT_EQ(umfDevDaxMemoryProviderParamsSetName(params_handle.get(), custom),
```

Comment:
you sometimes use ret, sometimes not - please be consistent

Reply:
it sooooo doesn't matter.... 

done